Packs/CyCognito/pack_metadata.json

```diff
@@ -7,9 +7,7 @@
     "url": "",
     "email": "support@cycognito.com",
     "categories": [
-        "Vulnerability Management",
-        "Data Enrichment & Threat Intelligence",
-        "External Exposure"
+        "Data Enrichment & Threat Intelligence"
```

Comment:
we dont need releaese notes for this pack?

Reply:
No need for RN when changing pack_metadata.
I ran `update-release-notes -g` to ensure everything gets updated.